src/styles/input-filters.scss

```diff
@@ -0,0 +1,3 @@
+.form-control {
+  background-color: rgba(159 194 232 / 20%) !important;
```

Comment:
If possible we should be avoiding `!important`. Instead you can make the rule more specific with extra classes